google/cloud/security/common/data_access/bucket_dao.py

```diff
@@ -14,8 +14,19 @@
 
 """Provides the data access object (DAO) for buckets."""
 
+from MySQLdb import DataError
+from MySQLdb import IntegrityError
+from MySQLdb import InternalError
+from MySQLdb import NotSupportedError
+from MySQLdb import OperationalError
+from MySQLdb import ProgrammingError
+
+from google.cloud.security.common.data_access import errors
 from google.cloud.security.common.data_access import project_dao
 from google.cloud.security.common.data_access.sql_queries import select_data
+# pylint: disable=line-too-long
+from google.cloud.security.common.gcp_type.bucket_access_controls import BucketAccessControls
```

Comment:
nit: import the module instead of the class name, e.g.

```
from google.cloud.security.common.gcp_type import bucket_access_controls as bkt_acls
```